acceptance_tests/features/steps/view_collection_exercise_created_state.py

```diff
@@ -0,0 +1,34 @@
+from behave import given, when, then
+
+from acceptance_tests.features.pages import collection_exercise, collection_exercise_details
+from controllers.collection_exercise_controller import get_survey_collection_exercises
+
+
+@given('the collection exercises for that survey exist')
+def ce_exist_for_survey(_):
+    bricks_id = 'cb8accda-6118-4d3b-85a3-149e28960c54'
+    collection_exercises = get_survey_collection_exercises(bricks_id)
+    assert len(collection_exercises) != 0
+
+
+@when('the internal user navigates to survey details page')
```

Comment:
Getting step not implemented due to a typo `the internal user navigates to survey details page` -> `the internal user navigates to the survey details page`
```
@when(u'the internal user navigates to the survey details page')
def step_impl(context):
    raise NotImplementedError(u'STEP: When the internal user navigates to the survey details page')
```

Reply:
Yeah, I had the same. Pushed up the changes now, @benjefferies.